Bottom: 29fc6c343092d9e7484a8109e2361018cbd3df68
Top:    55eae8334cba3a1c9184a6e9e2d5ec3ff1fd2a47
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:21:01 -0500

Add new host buildbot-worker.novatech-llc.com

Runs buildbot worker containers


---
diff --git a/ansible-playbook/production b/ansible-playbook/production
index 07414537..67202894 100644
--- a/ansible-playbook/production
+++ b/ansible-playbook/production
@@ -3,6 +3,10 @@ daedalus.novatech-llc.com
 
 [docker-hosts]
 daedalus.novatech-llc.com
+buildbot-worker.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[buildbot-worker]
+buildbot-worker.novatech-llc.com
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index e544fd42..6c319766 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -3,6 +3,10 @@ testdaedalus.novatech-llc.com
 
 [docker-hosts]
 testdaedalus.novatech-llc.com
+buildbot-worker.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[buildbot-worker]
+buildbot-worker.novatech-llc.com
